native/src/widget/radio.rs

```diff
@@ -196,8 +212,8 @@ where
             label_layout.bounds(),
             &self.label,
             self.text_size.unwrap_or(renderer.default_size()),
-            Default::default(),
-            None,
+            self.font,
+            self.text_color.clone(),
```

Comment:
Is calling `clone` necessary here? `Color` implements `Copy`.

Reply:
nope, I just typed unconsciously :)